libs/application/templates/passport/src/forms/Draft.ts

```diff
@@ -189,14 +189,13 @@ export const Draft: Form = buildForm({
               placeholder: m.dropLocationPlaceholder.defaultMessage,
               options: ({
                 externalData: {
-                  districtCommissioners: { data },
+                  deliveryAddress: { data },
                 },
               }) => {
                 return (data as DistrictCommissionerAgencies[])?.map(
-                  ({ id, name, place, address }) => ({
-                    value: id,
-                    label: `${name}, ${place}`,
-                    tooltip: `${address}`,
+                  ({ key, name }) => ({
+                    value: `${key}`,
+                    label: `${name}`,
```

Comment:
```suggestion
                    value: key,
                    label: name,
```
Is the template string needed?